src/runtime_src/core/pcie/driver/linux/xocl/subdev/ospi_versal.c

```diff
@@ -16,11 +16,11 @@
  */
 
 /*
- * This subdriver is used to transfer Firmware Image from host memory
- * to device memory through a dedicated BRAM. This BRAM is mapped to
- * mgmt function and both drivers running on host and device can access
- * this BRAM. The first 4 Bytes of the BRAM is the packet header and
- * others are data payload.
+ * This subdriver is used to transfer data, e.g. firmware image, XCLBIN
```

Comment:
Should we change the name of this file to xfer_versal.c?

Reply:
I am glad you ask. And yes, we'd better make it consistency between the file name, module name, variable pre-fix, etc, including the device side zocl driver too. While I am making the change from ospi-->xfer, when we decide to call the BRAM from OSPI cache to XFER cache, the name is being discussed again. xfer wil not be the final name. I will follow up with a name change after we finalized the ssv2 name for this BRAM.